camelot/view/mainwindow.py

```diff
@@ -124,6 +133,7 @@ def closeEvent( self, event ):
         self.write_settings()
         logger.info( 'closing mainwindow' )
         model_thread.stop()
-        super( MainWindow, self ).closeEvent( event )
+        #super( MainWindow, self ).closeEvent( event )
+        self.window.closeEvent( event )
         QtCore.QCoreApplication.exit(0)
```

Comment:
w dit closeEvent ooit gecalled ?

ik bedenk net dat er denk ik ook een mogelijkheid is in Qt, om een 'EventFilter' te injecteren in een bestaande widget, hiermee kan mss hetzelfde effect w bereikt, zonder die method te overschrijven

Reply:
Nee, die closeEvent w natuurlijk nooit gecalled. Ik had dit over het hoofd gezien.

Met een event filter werkt het wel (zie nieuwe commits).